models/user.schema.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require("mongoose");
+
+const UserSchema = mongoose.Schema;
+// Schema
+const userSchema = new UserSchema(
+  {
+    firstName: {
+      type: String,
+      required: true,
+    },
+    lastName: {
+      type: String,
+      required: true,
+    },
+    email: {
+      type: String,
+      required: true,
+      primaryKey: true,
```

Comment:
I don't think there is option in mongoose like `primaryKey`, It should be `unique: true`